src/Microsoft.Data.SqlClient/netcore/src/Resources/SR.Designer.cs

```diff
@@ -19,7 +19,7 @@ namespace System {
     // class via a tool like ResGen or Visual Studio.
     // To add or remove a member, edit your .ResX file then rerun ResGen
     // with the /str option, or rebuild your VS project.
-    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "15.0.0.0")]
+    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "16.0.0.0")]
```

Comment:
Please revert this change, as we don't intend to enforce VS 2019

Reply:
I don't think this means that VS 2019 is required, this just indicates the tool that last generated this file, but will revert.